tailwind.config.js

```diff
@@ -30,6 +30,10 @@ module.exports = {
       "base-400": "var(--base-400)",
       "base-500": "var(--base-500)",
     },
+    fontFamily: {
+      sans: ["Inter", "sans-serif"],
+      heading: ["Inter", "sans-serif"],
+    },
```

Comment:
This is nice to have uniformity, I let you know which good font we will be using in a moment

Reply:
I copy it from new theme repo.